scripts/go_executable_build.sh

```diff
@@ -50,6 +51,7 @@ OPTIONS:
    -b bucket      set the upload bucket name (default: $BUCKET)
    -f folder      set the upload folder name in the bucket (default: $FOLDER)
    -r             enable -race build option (default: $RACE)
+   -v             verbose build process (default: $VERBOSE)
```

Comment:
`VERBOSE` defaults to an empty string on line 16 above, so this line would be confusing:

```
   -v             verbose build process (default: )
```

Reply:
yes, it is default empty as to simplify the code.